ed_sensor_integration/include/ed/laser/updater.h

```diff
@@ -83,6 +82,7 @@ class LaserUpdater
     void renderWorld(const geo::Pose3D sensor_pose, const ed::WorldModel& world,
```

Comment:
Why does it need to be public?

Reply:
Currently because it is used in the test. In the future I would like to move this function somewhere else and make it not part of the class. But I consider that outside the scope of this PR